### PR TITLE
Test UTC time

### PR DESCRIPTION
**Pull request by snoopyjoe**
UTC time is off by 7hrs on some builds. Works on my test box but not sure about your setup.

### CODE REVIEW

**Comment by eracknaphobia** [comment]
Unfortunately no, it sets all my times back 3 hours than my airing time (EST).

**Comment by eracknaphobia** [comment]
This works for me, how about on your end?
```
def utc_to_local(utc_dt):
    offset = datetime.now() - datetime.utcnow()
    local_dt = utc_dt + offset
    return local_dt
```

**Comment by snoopyjoe** [comment]
That works on my boxes. I added a second using timedelta because there was a one second loss in the offset.

offset = datetime.now() - datetime.utcnow()
    local_dt = utc_dt + offset + timedelta(seconds=1)
    return local_dt

**Comment by eracknaphobia** [comment]
Ok, great!

File: resources/lib/ps_vue.py [hunk 1]
```diff
@@ -445,16 +445,17 @@ def get_stream(url, airing_id, channel_id, program_id, series_id, tms_id, title,
         listitem = xbmcgui.ListItem()
         listitem.setMimeType("application/x-mpegURL")
 
-    inputstreamCOND = str(json_source['body']['dai_method']) # Checks whether stream method is "mlbam" or "freewheel" or "none"
+    '''inputstreamCOND = str(json_source['body']['dai_method']) # Checks whether stream method is "mlbam" or "freewheel" or "none"
 
-    if inputstreamCOND != 'freewheel' and xbmc.getCondVisibility('System.HasAddon(inputstream.adaptive)'):#Inputstream doesn't seem to work when dai method is "freewheel"
```
Comment (eracknaphobia) [inline resources/lib/ps_vue.py:450]: Didn't realize you commented out the entire inputstream code. Why should we disable it for everyone if it doesn't work on one version of inputstream adaptive? It should be on the user to make sure they are running stable versions.

**Comment by snoopyjoe** [comment]
On the Kodi 18 nightlies Inputstream is always enabled. So anyone using Leia would not be able to play any streams.The Kodi 17 repo will probably be updated as well which will break the addon on Krypton as well. This is just a temp fix until I can see what Inputstream changed.

I can though try to add another condition that may only apply to Kodi 18. I'll test it now and if it works then I'll push a new commit.

**Comment by eracknaphobia** [comment]
I'm using 18 and it works fine. I think this is getting a little too complicated. If inputstream adaptive is turned on use it, if it doesn't work the user can then disable it. Otherwise we're locking everyone out of a feature just because a few are experiencing issues.

**Comment by snoopyjoe** [comment]
What version of inputstream are you using on Kodi 18? Maybe my setup is wrong.

**Pull request merged**